src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -267,6 +267,11 @@
             Arrays.asList(
                     new String[] {"comments?", "notes?"}));
 
+    private FieldValidator() {
+    // utility class
```

Comment:
Add one more level of indentation to both lines (so, 4 spaces).  Like how it's done in `Assumption.java`